gulpfile.babel.js

```diff
@@ -29,12 +30,17 @@ gulp.task('build:static', ['clean:client'], () =>
 //Convert ES6 code in all js files in src/js folder and copy to
 //build folder as bundle.js
 gulp.task('build:client', ['clean:client'], () => compileClientJS(['./src/client/javascripts/main.js'], 'main.js', './dist-client/javascripts'));
+gulp.task('build:clientcss', [], () => compileClientCSS('./src/client/stylesheets/style.css', './dist-client/stylesheets'));
 
 gulp.task('build:server', ['clean:server'], () => compileNodeJS('src/{server,shared}/**/*.js', './dist-server'));
 
-gulp.task('build', ['build:client', 'build:server', 'build:static']);
+gulp.task('build', ['build:client', 'build:clientcss', 'build:server', 'build:static']);
 
-gulp.task('build:test-client', ['clean:test'], () => compileClientJS(['./src/test/browser/index.js'], 'index.js', './dist-test/test/browser'));
+// gulp.task('build:test-client', ['clean:test'], () => compileClientJS(['./src/test/browser/components/map.js'], 'index.js', './dist-test/test/browser'));
+
+gulp.task('build:test-client', ['clean:test'], () => gulp.src(['src/test/.setup.js', 'src/test/browser/components/map.js'])  
```

Comment:
I also found that you are not compiling the codes. This breaks our convention (runtime codes should be in es5). The test code maybe used in other systems such as profiling, runtime code analysis. And we may test others than components like utils or library files. Please stick with an index file and use it as an entry point.

Sent from my iPhone 

> On Sep 12, 2016, at 12:03 AM, elegantuniv <notifications@github.com> wrote:
> 
> In gulpfile.babel.js:
> 
> >  
> > -gulp.task('build:test-client', ['clean:test'], () => compileClientJS(['./src/test/browser/index.js'], 'index.js', './dist-test/test/browser'));
> > +// gulp.task('build:test-client', ['clean:test'], () => compileClientJS(['./src/test/browser/components/map.js'], 'index.js', './dist-test/test/browser'));
> > +
> > +gulp.task('build:test-client', ['clean:test'], () => gulp.src(['src/test/.setup.js', 'src/test/browser/components/map.js'])  
> I changed specified component into 'src/test/browser/components/*.js' like this.
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>